### DISCUSSION TITLE
Update targets when new package version is installed?

### DISCUSSION THREAD

**Discussion by llrs** [ropensci/targets#695] [Ideas]
I'm already using renv with targets bu I'm using a package that is on ongoing development and I install it again frequently. I would like to make the targets that are affected by updates on the package. Is there any way to detect that? I could use some cue, but that wouldn't force the update of the target. 
Alternatively I probably could set target that checked this (package version and/or git commit) and make all the other targets dependable on this one, but probably this is something that might be worth supporting on targets. 
Thanks!

**Comment by wlandau** [comment]
I recommend having a look at https://books.ropensci.org/targets/packages.html#package-based-invalidation. You can use the `imports` and `packages` arguments of `tar_option_set()`.

**Reply by llrs** [reply]
Ah! Apologies! Should have read and search the excellent book before posting. Thanks!
